### PR TITLE
Add followers page

### PR DESCRIPTION
**Pull request by dooven**
Added followers page in `/users/username/followers `
Screenshot from 2020-07-27 19-29-35

Adding `application/json` to the Accept header returns a json 
Screenshot from 2020-07-27 20-11-57

Users will also have an option to hide the followers page to the public
Screenshot from 2020-07-27 20-13-13

### CODE REVIEW

File: handlers.go [hunk 1]
```diff
@@ -883,3 +886,55 @@ func (s *Server) DeleteHandler() httprouter.Handle {
...
+				ctx.Message = "Error loading profile"
+			}
+
+			s.render("error", w, ctx)
+
+			return
+		}
+
+		if !user.IsFollowersPubliclyVisible && !ctx.User.Is(user.TwtURL) {
+			s.render("404", w, ctx)
```
Comment (dooven) [inline handlers.go:920]: not sure if I should show a 404 here or show an error

Comment (prologic) [reply handlers.go:920]: Ahh good question. Is there a more valid RESTful response we can give?

**Comment by prologic** [comment]
This is very nice! I love it. Can you fix the conflict (_sorry I've been hacking all day!_) and I'll check this out locally and have a play!

**Comment by prologic** [comment]
> Can you fix the conflict

It's okay I fixed it _I hope_.

File: handlers.go (hunk 1 above)
Comment (prologic) [reply handlers.go:941]: I _think_ we should respond with a `401 Unauthorized` response here.

**Comment by dooven** [comment]
> It's okay I fixed it _I hope_.
Thanks for merging!

**Comment by prologic** [comment]
> > It's okay I fixed it _I hope_.
> 
> Thanks for merging!

You're welcome! Thanks for contributing! If you contribute another PR don't forget to add yourself to the [AUTHORS]/AUTHORS) file :)

**Pull request merged**